SamplePod/Check.swift

```diff
@@ -9,5 +9,224 @@
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+    print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+    print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+    print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+       print("dgsf")
+    
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: SamplePod/Check.swift;
  Line: 230;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)